### PR TITLE
Neo4j init 

### PR DESCRIPTION
**Pull request by loonydev**
Added scala impl. of Neo4j DB from java.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -81,6 +82,7 @@ lazy val allOld = (project in file("allOld"))
     scalatest,
     scalatestSelenium,
     moduleMysql,
+    moduleNeo4j,
```
Comment (LMnet) [inline build.sbt]: You don't need to modify `allOld` project. It's only for backward compatibility.

Comment (loonydev) [reply build.sbt]: Done

File: build.sbt [hunk 2]
```diff
@@ -116,6 +117,7 @@ lazy val allOld = (project in file("allOld"))
 
       val scalatestSeleniumEx = new ExcludeModule((scalatestSelenium/name).value)
       val moduleMysqlEx       = new ExcludeModule((moduleMysql/name).value)
+      val moduleNeo4jEx       = new ExcludeModule((moduleNeo4j/name).value)
```
Comment (LMnet) [inline build.sbt]: This change is redundant too. Sorry for not pointing to this explicitly.

File: build.sbt [hunk 3]
```diff
@@ -146,6 +148,9 @@ lazy val allOld = (project in file("allOld"))
           case moduleMysqlEx(e) =>
             e.copy(child = e.child :+ exclude(Dependencies.moduleMysql.value))
 
+          case moduleNeo4jEx(e) =>
+            e.copy(child = e.child :+ exclude(Dependencies.moduleNeo4j.value))
+
```
Comment (LMnet) [inline build.sbt]: And this too

Comment (loonydev) [reply build.sbt]: NP, something else?

Comment (LMnet) [reply build.sbt]: No, all good!

**Comment by dimafeng** [comment]
@loonydev thank you for the PR!

**Pull request merged**